x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -174,6 +184,21 @@ export const findCaseUserActions = async (
   };
 };
 
+export const getCaseUserActionsStats = async (
+  caseId: string,
+  signal: AbortSignal
+): Promise<CaseUserActionsStats> => {
+  const response = await KibanaServices.get().http.fetch<CaseUserActionsStats>(
```

Comment:
The type of `http.fetch` should be `CaseUserActionStatsResponse` as it returns a snake case response.